src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -448,15 +448,12 @@ def testImportSharedFile(self, otherCls):
                        otherCls=activeTestClassesByName)
 
         def testImportHttpFile(self):
-            # Prepare random file for import
             self.master.partSize = self.mpTestPartSize
-            srcUrl = 'https://raw.githubusercontent.com/BD2KGenomics/toil/master/Makefile'
-            srcHash = hashlib.md5(urllib2.urlopen(srcUrl).read()).hexdigest()
-            # test import
-            jobStoreFileID = self.master.importFile(srcUrl)
-            with self.master.readFileStream(jobStoreFileID) as f:
-                fileMD5 = hashlib.md5(f.read()).hexdigest()
-            self.assertEqual(fileMD5, srcHash)
+            srcUrl, srcHash = ('https://raw.githubusercontent.com/BD2KGenomics/toil/12c711ff91caa5d'
+                               '8fb37a072bad376f447faa797/Makefile',
```

Comment:
It's better to wrap strings on natural boundaries. For URLs, natural would mean before a slash.